tests/console/validate_lvm_raid1.pm

```diff
@@ -0,0 +1,121 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: RAID1 on LVM partition validation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "y2_module_consoletest";
+use testapi;
+use utils;
+use Test::Assert ':all';
+use scheduler 'get_test_data';
+use version_utils 'is_sle';
+use power_action_utils 'power_action';
+use Utils::Backends 'has_ttys';
+
+sub run {
+    my $self = shift;
+
+    select_console 'root-console';
+
+    my $config = get_test_data();
+    $config->{expected_num_devs} = scalar @{$config->{disks}};
+    # actual_num_devs is used to get the number of the disks in raid
+    # as we move in and out a disk during the test. This is because
+    # after the reboot sle15 can recover the missing disk and we need
+    # to compare with the expected_num_devs
+    my $actual_num_devs = $config->{expected_num_devs};
+    _check_lvm_partitioning($config);
+    _check_raid1_partitioning($config, $actual_num_devs);
+    _remove_raid_disk($config, $actual_num_devs);
```

Comment:
As we are not sure how to make it work on xen and need to investigate, what's about excluding part to remove raid disk on that platform and just check partitioning?

Reply:
i think i have something. you can check the VR for sle-12-SP5 svirt-xen-hvm @x86_64. Unfortunately i have to run again some of the other flavors. and if they dont fail it will be good to go

Reply:
I didn't get your last comment. For svirt-xen-hvm in SLE-15-SP2 VR is missing. If that one also works, you just need to exclude the removal part for `svirt-xen-pv` using setting `MACHINE` (if not, you can exclude those 3 cases by code). I believe that in that architecture is very tricky to investigate the problem, so the job will be failing or label it with a bug for a lot of time and does not make sense, I agree with Rodion's comment so you could make it green an create a research task for that part.